### PR TITLE
update1

### PR DESCRIPTION
**Pull request by ik0675**

### CODE REVIEW

File: test/resources/recipe_source.py [hunk 1]
```diff
@@ -53,21 +53,16 @@ def get(self):
...
         # setup and make request
-        req_obj = HttpRequestFactory.create('requests')
-        headers = {
-            'content-type': 'application/json',
-            'X-Yummly-App-ID': '9cce27e7',
-            'X-Yummly-App-Key': 'b13c741344519e5f89cb0edb7e8043f6'
-        }
+        req_obj = self.req_handler
         req_handler = RequestHandler(req_obj, redis_cache)
         res_tuple = req_handler.get(url, headers)
```
Comment (joe91e) [inline test/resources/recipe_source.py]: `global name 'headers' is not defined`. Do not bother passing the second argument headers. pass in None.
Headers will be set by the base.py already.

File: test/resources/base.py [hunk 2]
```diff
@@ -27,33 +24,25 @@ def __init__(self):
...
         self.req_obj = HttpRequestFactory.create('requests')
-        """
-        @ICK: try moving these headers to the config file!
-        """
-        headers = {
-            'content-type': 'application/json',
-            'X-Yummly-App-ID': '9cce27e7',
-            'X-Yummly-App-Key': 'b13c741344519e5f89cb0edb7e8043f6'
-        }
         self.req_obj.set_headers(headers)
```
Comment (joe91e) [inline test/resources/base.py]: `global name 'headers' is not defined.` use CFG_OBJ.get(<variable name here from config>)
you cannot take out the variable instantiation and continue to use the variable and make references to a non existing variable.

File: test/resources/base.py [hunk 3]
```diff
@@ -15,10 +15,7 @@
 and then remove these below four lines once they are no longer being
 referenced (no longer needed).
 """
-LIMIT = "maxResult"
-LIMIT_DEFAULT = 6
-OFFSET = "start"
-OFFSET_DEFAULT = 0
+
```
Comment (joe91e) [inline test/resources/base.py:9]: after `import json` line, add `from services import CFG_OBJ`

**Comment by joe91e** [comment]
First peer review failed. Please do test the endpoints by using the postman to hit the endpoints.
1) make your changes.
2) run `python app.py`
3) hit the endpoints you made changes on. in this case, `http://127.0.0.1:5000/recipes`,
with an example url of `http://127.0.0.1:5000/recipes?q=onion+soup`.
4) verify that your you are getting a valid response and no python error is thrown.

**Comment by joe91e** [comment]
apply the same changes for `class Recipe(baseResource)`. Once you have finished making your changes and verified that your code works, take out the instruction comments. (@ick comments)

File: test/constants/__init__.py [hunk 4]
```diff
@@ -1 +1,7 @@
+import constants
```
Comment (joe91e) [inline test/constants/__init__.py:1]: you don't need this import statement.

**Comment by ik0675** [comment]
I will make changes :) 👍

**Comment by joe91e** [comment]
LGTM (looks good to me)

**Pull request merged**